src/Components/App/App.js

```diff
@@ -53,31 +53,19 @@ const App = () => {
     }
   };
 
-  const addUserDetails = (detailProperty, pokemonId, detailContents) => {
+  const togglePokemonFavoriteStatus = (pokemonId) => {
     const pokemonInfoModifier = (updatedPokemons, pokemon) => {
       if (pokemon.id !== pokemonId) {
         updatedPokemons = [...updatedPokemons, pokemon];
       } else {
-        pokemon[detailProperty] = detailContents;
+        pokemon.isFavorite = !pokemon.isFavorite;
         updatedPokemons = [...updatedPokemons, pokemon];
```

Comment:
Since we have the same line twice(62 and 57) I believe we could take it out of the if/else statement